curvefs/devops/inventory/server.ini

```diff
@@ -0,0 +1,2 @@
+[metaserver]
+localhost ansible_ssh_host=pubbeta2-curve7.dg.163.org ansible_ssh_port=1046
```

Comment:
remove our hostinfo and port num, use fake info.

Reply:
> remove our hostinfo and port num, use fake info.

The PR is `work in progress`, it has removed in the final PR.